src/test/java/com/easypost/ShipmentTest.java

```diff
@@ -403,6 +426,62 @@ public void testInstanceLowestSmartRate() throws EasyPostException {
         assertThrows(EasyPostException.class, () -> {
             vcr.client.shipment.lowestSmartRate(shipment.getId(), 0, SmartrateAccuracy.Percentile90);
         });
+
+        // Test deprecated smart rate filter function
+        Smartrate deprecatedLowestSmartRateFilters = vcr.client.shipment.lowestSmartRate(shipment.getId(), 2,
+                "percentile_90");
+
+        // Test lowest smartrate with valid filters
+        assertEquals("Priority", deprecatedLowestSmartRateFilters.getService());
+        assertEquals(8.15, deprecatedLowestSmartRateFilters.getRate(), 0.01);
+        assertEquals("USPS", deprecatedLowestSmartRateFilters.getCarrier());
+
+        // Test lowest smartrate with invalid filters (should error due to strict
+        // delivery days)
+        assertThrows(EasyPostException.class, () -> {
+            vcr.client.shipment.lowestSmartRate(shipment.getId(), 0, SmartrateAccuracy.Percentile90);
+        });
+    }
+
+    /**
+     * Test getting smart rates for a shipment.
+     * 
+     * @throws EasyPostException
+     */
+    @Test
+    public void testGetSmartRate() throws EasyPostException {
+        vcr.setUpTest("get_smartrate_list");
+        // Test deprecated getSmartrates() function
```

Comment:
These comments shouldn't be needed since the test name/logic would already explain this. Once the deprecated function gets removed, we'd simply remove these tests because they'd start failing once the logic gets ripped out.

TLDR: It's more clutter than provides value

Reply:
Removed